src/yb/master/backfill_index.h

```diff
@@ -350,8 +355,10 @@ class BackfillChunk : public RetryingTSRpcTask {
   std::string type_name() const override { return "Backfill Index Table"; }
 
   std::string description() const override {
-    return yb::Format("Backfilling index_ids $0 for tablet $1 from key '$2'",
-                      indexes_being_backfilled_, tablet_id(),
+    std::string requested_index_names = "";
```

Comment:
you could store this as a member variable in BackfillChunk, instead of recomputing it each time we call description.